demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -960,6 +959,7 @@ def test_generate_script_doc(tmp_path, mocker):
         IDSetCreator, "create_id_set", return_value=[id_set, {}, {}]
     )
     mocker.patch.object(common, "execute_command", side_effect=handle_example)
+    mocker.patch("os.path.isfile", return_value=False)
```

Comment:
why?

Reply:
For some reason it open the file, some things related to the env in circleCi